src/components/pages/Chat/MessageTransmitter.tsx

```diff
@@ -70,7 +72,13 @@ export default function MessageTransmitter({ children }: Props) {
                     }
 
                     await (async () => {
-                        const stream = await streamrClient.getStream(roomId)
+                        const metamaskStreamrClient = new StreamrClient({
+                            auth: { ethereum: ethereumProvider as any },
+                        })
```

Comment:
This MM streamrClient is used often. Maybe we should throw it into the global, state alongside `ethereumProvider`?

Reply:
agreed, i'll replicate `ethereumProvider`'s schema with the `metamaskStreamrClient`